android/src/main/java/com/frontegg/android/AuthenticationActivity.kt

```diff
@@ -103,13 +120,19 @@ class AuthenticationActivity : Activity() {
         private const val AUTH_LAUNCHED = "com.frontegg.android.AUTH_LAUNCHED"
         private const val CUSTOM_TAB_LAUNCHED = "com.frontegg.android.CUSTOM_TAB_LAUNCHED"
         private val TAG = AuthenticationActivity::class.java.simpleName
+        var onAuthFinishedCallback: (() -> Unit)? = null // Store callback
```

Comment:
Did you use this callback? You never fill this property only a safe call

Reply:
invoked in the invokeAuthFinished